cmd/internal/cli/verify.go

```diff
@@ -141,44 +136,64 @@ var VerifyCmd = &cobra.Command{
 }
 
 func doVerifyCmd(ctx context.Context, cmd *cobra.Command, cpath, url string) {
+	id, isGroup, err := ensureImageFlags(cmd, cpath, sifDescID, sifGroupID, verifyAll)
+	if err != nil {
+		sylog.Fatalf("%s", err)
+	}
+
+	// Dont need to resolve remote endpoint.
+	if !localVerify {
+		handleVerifyFlags(cmd)
+	}
+
+	author, _, err := signing.Verify(ctx, cpath, url, id, isGroup, verifyAll, authToken, localVerify, jsonVerify)
+	fmt.Printf("%s", author)
+	if err == signing.ErrVerificationFail {
+		sylog.Fatalf("Failed to verify: %s", cpath)
+	} else if err != nil {
+		sylog.Fatalf("Failed to verify: %s: %s", cpath, err)
+	}
+	sylog.Infof("Container verified: %s", cpath)
+}
+
+// ensureImageFlags takes a container path, sif id and group id, and formats to a more usable output.
+func ensureImageFlags(cmd *cobra.Command, cpath string, descrID, groupID uint32, all bool) (uint32, bool, error) {
```

Comment:
Would `ensureImageFlags` be better called something like `sifIDFromFlags` or similar? `ImageFlags` is a little generic to me.

Reply:
I changed it to `checkImageAndFlags`, and updated the comment. Does that sound a little better?